mmv1/third_party/terraform/tests/resource_storage_bucket_object_test.go

```diff
@@ -426,6 +446,35 @@ func testAccCheckGoogleStorageObjectWithEncryption(t *testing.T, bucket, object,
 	}
 }
 
+func testAccCheckGoogleStorageFolder(t *testing.T, bucket, folder string) resource.TestCheckFunc {
+	return func(s *terraform.State) error {
+		config := GoogleProviderConfig(t)
+
+		objectsService := storage.NewObjectsService(config.NewStorageClient(config.UserAgent))
+
+		getCall := objectsService.Get(bucket, folder)
+		if customerEncryptionKey != "" {
```

Comment:
Not sure why this is needed? 

A simple get should suffice. Credentials should be handled by the client.

Reply:
Ahh I see this is copy pasted from `testAccCheckGoogleStorageObjectWithEncryption`. Yeah you can remove this whole if statement and the enclosed header stuff.

Reply:
thank you. fixed